src/libproc.rs

```diff
@@ -0,0 +1,19 @@
+//! This module roughly corresponds to `libproc.h`.
```

Comment:
I'm not familiar with this, how does this relate to mach? It'd be great if it's documented.